app/api/chemotion/report_api.rb

```diff
@@ -127,16 +132,73 @@ class ReportAPI < Grape::API
       params do
         requires :ids
         requires :settings
+        requires :configs
       end
 
       get :rtf do
         ids = params[:ids].split("_")
-        settings = params[:settings].split("_")
-        rtf_data = Template::ReactionsReport.new(ids, settings).get_rtf_data
+        settings = set_settings(params[:settings].split("_"))
+        configs = set_configs(params[:configs].split("_"))
+        reactions = ids.map { |id| Reaction.find(id) }
+        contents = Report::Docx::Document.new(reactions).process.contents
+
+        filename = "ELN_Reactions_" + Time.now.strftime("%Y-%m-%dT%H-%M-%S") + ".docx"
+        template_path = File.expand_path("../../../../lib/template", __FILE__) + "/ELN_Reactions.docx"
```

Comment:
if you move `report_api.rb` file to somewhere else this will stop working.
not so important but maybe try to use Rails.root path